packages/app/src/NodeDetailsSidePanel.tsx

```diff
@@ -0,0 +1,286 @@
+import * as React from 'react';
+import Link from 'next/link';
+import Drawer from 'react-modern-drawer';
+import { StringParam, useQueryParam, withDefault } from 'use-query-params';
+import {
+  Anchor,
+  Badge,
+  Card,
+  Flex,
+  Grid,
+  SegmentedControl,
+  Text,
+} from '@mantine/core';
+
+import { DrawerBody, DrawerHeader } from './components/DrawerUtils';
+import api from './api';
+import {
+  convertDateRangeToGranularityString,
+  K8S_CPU_PERCENTAGE_NUMBER_FORMAT,
+  K8S_MEM_NUMBER_FORMAT,
+} from './ChartUtils';
+import HDXLineChart from './HDXLineChart';
+import { InfraPodsStatusTable } from './KubernetesDashboardPage';
+import { LogTableWithSidePanel } from './LogTableWithSidePanel';
+import { parseTimeQuery, useTimeQuery } from './timeQuery';
+import { formatUptime } from './utils';
+import { useZIndex, ZIndexContext } from './zIndex';
+
+import styles from '../styles/LogSidePanel.module.scss';
+
+const CHART_HEIGHT = 300;
+const defaultTimeRange = parseTimeQuery('Past 1h', false);
+
+const PodDetailsProperty = React.memo(
+  ({ label, value }: { label: string; value?: React.ReactNode }) => {
+    if (!value) return null;
+    return (
+      <div className="pe-4">
+        <Text size="xs" color="gray.6">
+          {label}
+        </Text>
+        <Text size="sm" color="gray.3">
+          {value}
+        </Text>
+      </div>
+    );
+  },
+);
+
+const NodeDetails = ({
+  name,
+  dateRange,
+}: {
+  name: string;
+  dateRange: [Date, Date];
+}) => {
+  const where = `k8s.node.name:"${name}"`;
+  const groupBy = ['k8s.node.name'];
+
+  const { data } = api.useMultiSeriesChart({
+    series: [
+      {
+        table: 'metrics',
+        field: 'k8s.node.condition_ready - Gauge',
+        type: 'table',
+        aggFn: 'avg',
```

Comment:
probably want `last_value` for both of these

Reply:
fixed